core-processor/Cargo.toml

```diff
@@ -15,6 +15,7 @@ gear-core.workspace = true
 gear-core-errors = { workspace = true, features = ["codec"] }
 gear-backend-common.workspace = true
 gear-wasm-instrument.workspace = true
+gear-lazy-pages-common.workspace = true
```

Comment:
@breathx and @NikVolf  does not wanna have substrate deps in core-processor. So, you need to discuss this with them first

Reply:
This is interesting. Because with lazy-pages always enabled we need to have substrate deps in all crates which uses core-processor or you need to make separate lazy-pages realisation.